tests/Observatory/test_ObservatoryL2Halo.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+from tests.TestSupport.Info import resource_path
+import EXOSIMS.Observatory.ObservatoryL2Halo as obs
+import pkgutil, os, json, sys, copy
+from EXOSIMS.util.get_module import get_module
+import numpy as np
+from astropy.time import Time
+import astropy.units as u
+from tests.TestSupport.Utilities import RedirectStreams
+
+# Python 3 compatibility:
+if sys.version_info[0] > 2:
+    from io import StringIO
+else:
+    from StringIO import StringIO
+
+class TestObservatoryL2Halo(unittest.TestCase):
+    """
+
+    ObservatoryL2Halo test.
+    Made specifically (for now) to test the jacobian_CRTBP calculation. 
+
+    """
+
+    def setUp(self):
+
+        self.fixture = obs.ObservatoryL2Halo()
+
+    def tearDown(self):
+      del self.fixture
+
+    def test_jacobian_CRTBP(self): 
+        """
+        Strategy: Make sure the output is of the right type/size. Arg 't' doesn't
+        seem to do anything yet in ObservatoryL2Halo. 
+        """
+
+    #     Z = np.zeros([3,3,1])
+    #     E = np.full_like(Z,np.eye(3).reshape(3,3,1))
```

Comment:
I cannot merge code with entire functions commented like this. The function needs to do something.

Reply:
I think I'll remove this test for now. I attempted calculating the Jacobian function myself via Gabe's thesis and I got a different Hessian component to the final matrix. 

I might start playing around with branches more so you don't see messy things like this.